### PR TITLE
correct installation directory for Fortran module files.

### LINKED ISSUE #129: include install directory destination is incorrect in cmake.
`module_dir` variable for Fortran module files in [L189](https://github.com/NOAA-EMC/EMC_post/blob/564e26f550d5b444a885286fe6e648249021ed04/sorc/ncep_post.fd/CMakeLists.txt#L189) is  defined as:
```
set(module_dir "${CMAKE_CURRENT_BINARY_DIR}/include")
```

During install, these files are copied into the destination `include` as seen in [L221](
https://github.com/NOAA-EMC/EMC_post/blob/564e26f550d5b444a885286fe6e648249021ed04/sorc/ncep_post.fd/CMakeLists.txt#L221) reproduced here as:
```
install(DIRECTORY ${module_dir} DESTINATION include)
```

This means that the final installation directory containing these Fortran module files is:
```
${CMAKE_INSTALL_PREFIX}/include/include
```

The correct [L221](https://github.com/NOAA-EMC/EMC_post/blob/564e26f550d5b444a885286fe6e648249021ed04/sorc/ncep_post.fd/CMakeLists.txt#L221) should be
```
install(DIRECTORY ${module_dir} DESTINATION ${CMAKE_INSTALL_PREFIX})
```

### PR DESCRIPTION
**Pull request by aerorahul**
This PR fixes the installation path of compiled Fortran module files.

Previously their destination was:
 `${CMAKE_INSTALL_PREFIX}/include/include`

This has now been corrected to:
`${CMAKE_INSTALL_PREFIX}/include`

closes #129

### CODE REVIEW

**Review by fossell — APPROVED** [review]
This change only impacts cmake builds, no impact to unipost functions.  I can merge once approval given by another release team member.

**Review by WenMeng-NOAA — APPROVED** [review]

**Review by DusanJovic-NOAA — APPROVED** [review]

**Pull request merged**
